### PR TITLE
Update Tangle according audit/review

### LINKED ISSUE #1137: [CHECKLIST] Review/audit Tangle website UI and copy
## Overview

This task will outline a series of smaller tasks focusing on improvements, fixes, and additions to the Tangle website. The dApp team implemented each section of the Tangle site last sprint, and require a thorough review of the UI components and we require some copy in specific sections. 

#### Development Link
- [Tangle Dev ](https://tangle-webbsite.netlify.app/)

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes
Update the Tangle Website according audit/review

### Proposed area of change
_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/stats-dapp`
- [ ] `apps/webbsite`
- [ ] `apps/faucet`
- [x] `apps/tangle-website`
- [x] `libs/webb-ui-components`

### Reference issue to close (if applicable)
Closes #1137 

### Screen Recording
_If possible provide a screen recording of proposed change._

-----
### Code Checklist 
_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by AtelyPham — COMMENTED** [review]
The code LGMT ✅ Please wait for @monaiuu and @thomivy to review the UI updates and copies before merging.

**Comment by monaiuu** [comment]
Thanks for the quick updates!! Margins for hero section LGTM! 

**Found a few more UI updates that'll be nice to include before merging:**

Screenshot 2023-05-10 at 8 10 32 AM

- [ ] Hero image becomes distorted between ~1440px -> 1024px

Screenshot 2023-05-10 at 8 32 09 AM

- [ ] Remove the shadow behind the annotation component
- [ ] Add hover state to the annotation component when the tooltips open


Screenshot 2023-05-10 at 8 13 12 AM

- [ ] Features section cards images become distorted going into smaller breakpoints (under 768px)

**Comment by dutterbutter** [comment]
@vutuanlinh2k2 please address changes requested here: https://github.com/webb-tools/webb-dapp/pull/1175#issuecomment-1542433471

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**
